code/modules/procedural mapping/planetGenerator/story_generator/clown.dm

```diff
@@ -0,0 +1,290 @@
+/datum/story_theme/clown
+	name = "clown"
+	theme_flag = STORY_CLOWN
+	hostile_mobs = list(
+		/mob/living/simple_animal/hostile/retaliate/clown
+	)
+	corpse_types = list(
+		/obj/effect/landmark/corpse/clown
+	)
+
+/datum/story_theme/clown/generate_character_name()
+	if(clown_names?.len)
+		character_name = pick(clown_names)
+	else
+		character_name = "Sprinkles the Clown"
+	return character_name
+
+/datum/story_theme/clown/get_log_info()
+	return list("title" = "THE HONKENING CONTINUES", "subtitle" = "A Clown's Tale by [character_name]")
```

Comment:
Add a couple clownified log names.
"The banana peel stories"
"how i learned to sotp worrying and love the HONK"
"hey AI this is the clonw, part 1"
etc